src/routes/blog/posts/v3-4-1/+page.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Announcing Files v3.4.1
+description: A notification when app is running in the background, prompt when failing to rename items, and improvements to stability.
```

Comment:
This appears to be a description of an earlier version.

Reply:
Thanks